### DISCUSSION TITLE
Aqara temperature/humidity sensor bricked

### DISCUSSION THREAD

**Discussion by gkgeorgiev** [Koenkk/zigbee2mqtt#17496] [Devices]
Has anyone experienced dead Aqara sensors? In my case, i had two Aqara temperature/pressure/humidity sensors but they both stopped responding completely. I've checked the batteries and tried with new ones, but without any luck. No signs of life and they've been working for about two years. 
My two theories are - either hardware got damaged due to high humidity (bathroom) or some firmware update that bricked them ( don't remember if i triggered any updates though).

**Comment by lux73** [comment]
have you tried to delete (force) the Sensors from z2m and completely new pairing it?

**Reply by gkgeorgiev** [reply]
Yeah, this was one of the first things i tried...but the problem is that the sensors don't blink at all when trying to reset them.

**Reply by lux73** [reply]
then i would assume they are defect - last thing you could try is to replace the battery ^^

**Reply by gkgeorgiev** [reply]
Yeah...i did that already... Probably have defected....but what puzzled me is that they've been working for about 2 years and they died at approximately the same time 🤔

**Reply by lux73** [reply]
very strange two devices dead same time ^^

**Reply by Shred99** [reply]
Take the battery out, short across the battery contacts to drain any capacitor that may be in parallel with the battery, then leave the battery out for a day or two.  Try re-pairing it then.

I've had some of the newer Aqara sensors appear to die and not come back after changing the battery, re-pairing etc etc.  Completely resetting them as a above has fixed it.

**Reply by sampatz07** [reply]
how did you do this?

**Comment by gkgeorgiev** [comment]
All of the above didn't work for me 😔 ..now i have 3-4 bricked sensors

**Reply by scottgasch** [reply]
Open the battery panel and look at the battery contacts.  Check whether the one in the middle is sticking up enough and, if not, bend it a little.  The one on the side should be in the recess and not bent over -- if it's bent over it will contact the front of the watch battery and not the side.  In my case the latter was the problem -- I must have somehow bent the contact when messing with getting the sensor pulled off the surface where it was sticky-taped up.  Once I fixed the contacts the sensor "unbricked" itself.  Hope this helps, good luck.

**Reply by magnetgrouplabs** [reply]
> Open the battery panel and look at the battery contacts. Check whether the one in the middle is sticking up enough and, if not, bend it a little. The one on the side should be in the recess and not bent over -- if it's bent over it will contact the front of the watch battery and not the side. In my case the latter was the problem -- I must have somehow bent the contact when messing with getting the sensor pulled off the surface where it was sticky-taped up. Once I fixed the contacts the sensor "unbricked" itself. Hope this helps, good luck.

I know this comment was months ago, but this just saved me from buying a replacement sensor. The battery contact on the side was bent for mine also after I replaced the battery. Bending it back got my sensor working again. Thank you!!!

**Reply by jdesigndev** [reply]
> Open the battery panel and look at the battery contacts. Check whether the one in the middle is sticking up enough and, if not, bend it a little. The one on the side should be in the recess and not bent over -- if it's bent over it will contact the front of the watch battery and not the side. In my case the latter was the problem -- I must have somehow bent the contact when messing with getting the sensor pulled off the surface where it was sticky-taped up. Once I fixed the contacts the sensor "unbricked" itself. Hope this helps, good luck.

It worked for me. THKS

**Comment by gkgeorgiev** [comment]
So far, I've tried the following

1. Replacing the battery
2. bending the metal plates for better battery contact
3. hard-resetting by pressing for 10-15 seconds while inserting the batter
4. trying to empty the capacitors by keeping the reset button pressed, without a battery
5. shortcut of the two battery connectors
6. And yes - it's not the humidity since all except one are in my living room and Zurich climate is not that humid

Non of these worked. 
The weird thing is that the same happened to sensors of different providers - Aqara (temperature/pressure/..) and Tuya (PIR and light sensor). Also I'm not a low-voltage specialist, but wondering if the Wifi 2.4/5GHz  networks or my 5G router could somehow interfere or induce some electricity that could also fry the circuits. So far there are 5 sensors dead and I start wondering  whether they could be somehow fried by the protocol (overloading some sort) So far it's sensors only. All power connected zigbee devices are working perfectly.

**Reply by rudelm** [reply]
seems unlikely. But interesting that others are having this issue as well with their Aqara sensors. Out of 10, one is now completely dead. I don't think its a Zigbee2Mqtt problem though but rather a hardware problem.

**Comment by gkgeorgiev** [comment]
I'm glad that at least for someone it has a happy ending 😄

**Comment by ghost** [comment]
i have one that stopped doing anything besides drawing a constant 2.07mA. it's the one that was measuring outside values. i originally thought it's just the battery again (this one went through batteries faster than the others), but it didn't come back to life with good, new varta cells. it just drained them. so i powered it from a bench supply and started probing. the oscillator is working. nice sine wave at around 32MHz. but the constant 2mA draw tells me something's off badly. this is not a case of bent contacts...

Optically it looks like new. no traces of water

**Comment by henrikdalgaard** [comment]
Two dead TVOC sensors here - 5 months between. Seems related to running out of battery

**Reply by newbyp** [reply]
I've got the Aqara temperature and humidity sensor and it stops reporting. Flatlined so I changed the battery and thought it was alive but it flatlined again a little later. I assume it's dead. Luckily I have a spare to try in its place.
